distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/axis2/axis2.xml.j2

```diff
@@ -565,7 +565,7 @@
         <phase name="StatReporting"/>
     </phaseOrder>
 
-    <clustering class="org.wso2.carbon.core.clustering.hazelcast.HazelcastClusteringAgent"
+    <clustering class="org.wso2.carbon.hazelcast.HazelcastClusteringAgent"

```

Comment:
lets template this class and put a default class to this without hazelcast

Reply:
Templated the class with https://github.com/wso2/carbon-kernel/pull/3398/commits/a288d4d1e067b7a49e6ff30409c6fe6ebb518020